e2e/tests/test_e2e.py

```diff
@@ -587,6 +587,27 @@ def verify_role():
             print('Operator log: {}'.format(k8s.get_operator_log()))
             raise
 
+    @timeout_decorator.timeout(TEST_TIMEOUT_SEC)
+    def test_cross_namespace_secrets(self):
+        '''
+            Test secrets in different namespace
+        '''
+        app_namespace = "appspace"
+        k8s = self.k8s
+        k8s.api.core_v1.create_namespace(app_namespace)
```

Comment:
I am unsure if `kind` always creates a new namespace fast enough. It is better to have some type of timeout / waiting here to avoid spurious test failures.